.env_SAMPLE

```diff
@@ -72,7 +73,7 @@ export SHEER_ELASTICSEARCH_INDEX=content
 
 # Name of the virtualenv created for Sheer in
 # https://github.com/cfpb/cfgov-refresh/blob/flapjack/INSTALL.md#sheer
-export VIRTUAL_ENV=cfgov-refresh
+export VENV_NAME=testing
```

Comment:
`VIRTUAL_ENV` is set automatically when activating a virtualenv, so we should use a unique name.

Also, now remembering that I should probably set this back to cfgov-refresh before merging this.